### PR TITLE
Clarify bundle naming convention

### LINKED ISSUE #107: Need better clarity on bundle filename for chains of attestations
The current bundle [file naming convention](https://github.com/in-toto/attestation/blob/main/spec/bundle.md#file-naming-convention) is misleading for the case when the bundle is a set of dependent attestations that are all "about" a single artifact but have different subjects. Current wording:

> -   Bundles that concern a single artifact SHOULD name the bundle file `<artifact filename>.intoto.jsonl`.
> -   Bundles that concern multiple artifacts SHOULD name the bundle file `multiple.intoto.jsonl`.

The phrase "concern a single artifact" needs more specificity. It could be read two ways:

- (unintended) If any of the attestations in the bundle have a subject other than `<artifact>`, name it `multiple.intoto.jsonl`.
- (intended) If the set of attestations are intended to verify a single artifact, regardless of the attestations' subject, name it `<artifact filename>.intoto.jsonl`.

cc: @shaunmlowry @TomHennen @mlieberman85

### PR DESCRIPTION
**Pull request by MarkLodato**
Clarify the naming convention to be unambiguous when a single bundle "concerns" a single artifact but has attestations for other dependent artifacts.

Remove the recommendation for `multiple.intoto.jsonl`. In its absence, implementations can choose a name that makes sense for that use case.

Fixes #107.

### CODE REVIEW

**Review by TomHennen — APPROVED** [review]
This looks good to me.  Might be good to get a second reviewer before merging.

[edit]: I guess I don't have write access (surprising!) so you need another reviewer anyways.

**Review by laurentsimon — APPROVED** [review]
LGTM

**Review by marcelamelara — APPROVED** [review]
LGTM, will merge.

**Comment by marcelamelara** [comment]
@TomHennen Regarding write access, you don't seem to be part of the maintainers team yet (though you should be).

cc @adityasaky @SantiagoTorres

**Comment by adityasaky** [comment]
@TomHennen, can you accept the invitation to the attestation maintainers team?

**Pull request merged**
